httpfuzzerprocessor/RTT_time_filter.py

```diff
@@ -0,0 +1,44 @@
+# Version 1.0
+# @author RUFFENACH Timothée
+# filter by RTT (time request).
+
+from javax.swing import JFrame, JPanel, JComboBox, JOptionPane,JFileChooser,JOptionPane
+
+# Auxiliary variables/constants needed for processing.
+global time,isCheck;
+init = False
+
+def getNumber(min,max,asked):
+	number = JOptionPane.showInputDialog(None, asked, "Input", JOptionPane.QUESTION_MESSAGE)
+    
+	if int(number) >= min and int(number) <= max:
+		number = int(number)
+		return number
+	else:
+		JOptionPane.showMessageDialog(None, "Choose number between " +  min  + " to " + max)
+		getNumber()
+
+# Called after injecting the payloads and before forward the message to the server.
+def processMessage(utils, message) :
+	if (init == False):
+		initialise()
```

Comment:
Maybe "shouldInit" should be more clear to future users/maintainers?

Reply:
ok Im have rename to shouldInit